docs/api.md

```diff
@@ -118,6 +118,20 @@ Writing formats that cannot represent all aspects of {class}`AnnData` objects.
    AnnData.write_loom
 ```
 
+(utilities-api)=
+
+## Utilities
+
+Helper functions used internationally or for reshaping and aligng `AnnData` objects. Can be useful for cusotm workflows or edge cases.
+
+```{eval-rst}
+.. autosummary::
+	:toctree: generated/
+
+	utils.adapt_vars_like
+
+```
+
```

Comment:
Yup this looks great!!!

Reply:
The little `(utilities-api)=` at the top helps with intersphinx mapping i..e, accessing sections of docs from one project in another project's docs.  Thanks for adding it!